frontend/src/app.css

```diff
@@ -48,3 +48,73 @@
     columns: 5rem auto;
   }
 }
+body {
+  height: calc(100vh - 108px) !important;
+  overflow: auto;
+}
```

Comment:
But you can see the view on mobile looks good, because i have used the breakpoints for that. 
Sure i will remove the overflow property

Reply:
Yes I see the breakpoint, but what about hiding the player? I guess current approach will not work in that case. Also, I feel some doubt in using pixels here. Imagine that the user will have different scale factor or font size… I believe the layout will mess up then. That is why Tailwind almost everywhere uses `rem`.

Try using vertical flex, as I used it in the filter panel for mobiles.

Reply:
how which event i can remove the hidden attribute from the div?

Reply:
I can't fully understand the question.

What I mean is that the web page should look correctly with the audio player shown and hidden.

You don't need to manipulate with the attribute yourself, my code will do that.

Reply:
I will place the `hidden` attribute on the `player div`, you just need to remove it when you don't need it.

Reply:
or you just need to set the hidden attribute true or false. Like this 
```
document.getElementById("player").hidden = true;
document.getElementById("player").hidden = false;
```

Reply:
> I will place the hidden attribute on the player div, you just need to remove it when you don't need it.

Exactly, that's how it will work. Actually, you can keep it shown in the code, but make sure the page looks good with it hidden.